### ISSUE TITLE
Announcing Nervos Grant - Blog post

### ISSUE THREAD

**Issue by zschavi** [leapdao/meta#248]
# Bounty
Serious Grant was accepted from outside the Ethereum Ecosystem.
LeapDAO can help achieve scalability in diverse environments. Let's share the news and tell the world what we do for Nervos Network.

## Scope

explain/include/mention:
- Ethereum is the home of DApps, but has it's flaws (scalability & ux trouble showed up late)
- nervos recognizes this and offers layer2 support natively
- our grant was approved to create a ethereum compatible layer2 on Nervos
- aligned interests & shared thoughts led to a great mashup of skills & incentives
- call to action for dev's & contributors - who's in to support the project ?

## Deliverables

- blog post
- tl;dr for twitter (will be posted as thread, w/ max 8 tweets)

## Funding Circle

general circle
## Bounty Owner/Gardener

@zschavi  as **Social Maintainer**

## Gain for the Role

increased reach for org, attracting potential contributors
## Roles
bounty gardener: @zschavi / 100 DAI
bounty worker: @Clearwood / 350 DAI
bounty reviewer: @johannbarbie / 100 DAI

## Gardener checklist

- [x] I have attached one of the `size-` labels to this bounty
- [x] I have submitted this bounty via http://bounty.leapdao.org/viewform
- [x] I have added this bounty to bounty board here https://github.com/orgs/leapdao/projects/6

**Comment by johannbarbie** [comment]
some related ideas by ben
<img width="690" alt="Screen Shot 2020-06-15 at 4 38 14 PM" src="https://user-images.githubusercontent.com/659301/84771458-c4a2f780-afd9-11ea-8691-3661b7492d88.png">
